### ISSUE TITLE
Need for windows machine

### ISSUE THREAD

**Issue by msva** [mandl/LinkIt_Assist_2502#1]
Hi!
You're mentioning

> Download the Windows SDK
> Use a Windows maschine for updating the firmware

in "setup" section of the README. However, it seems it **is** a [way](http://labs.mediatek.com/site/global/developer_tools/mediatek_linkit_assist_2502/api_references/vm_firmware_trigger_update.gsp) to upgrade the firmware without Windows host.

Although, documentation says

> Calling this function causes the system to reboot immediately and then trigger the firmware update process in bootloader to update full firmware image. The path to the image must be "C:\image.bin". 

While, in _my_ case, it is a bit unclear, what exactly should I flash on my RePhone (which seems to be the clone(?) of 2502): exactly `image.bin` from firmware directory, and which has the size of 4.7Mb, or the long-named `.bin` file with SEEED in the name (and the size of 7Kb), since it is mentioned in `SEEED*.cfg` (which, in turn, mentioned on the RePhone wiki).

So, TL;DR:
I think, it whould be working way to upgrade firmware by copying `image.bin` in the root of MassStorage device, and uploading vxp sketch with something like 

```
#include "vmfirmware.h"
int main() { vm_firmware_trigger_update(); }
```

P.S. Haven't you any idea, how to "catch" `ttyUSB0` to not disappear? :) It seems, Windows Flasher doing exactly that: catching it by sending some magic string there, so device doesn't activate Mass Storage Mode and listen for commands on `ttyUSB0`, and I think, that's why it asks to poweroff the device and connect it **after** pressing "upload firmware".

**Comment by mandl** [comment]
at moment. i am doing some research .....
how we can manage the update.
